mkosi/__init__.py

```diff
@@ -968,22 +968,24 @@ def make_tar(state: MkosiState) -> None:
         run(cmd)
 
 
-def find_files(root: Path) -> Iterator[Path]:
+def find_files(dir: Path, root: Path) -> Iterator[Path]:
     """Generate a list of all filepaths relative to @root"""
```

Comment:
The docstring is wrong with this change.

Reply:
well spotted!